src/seedu/addressbook/commands/Command.java

```diff
@@ -45,6 +45,8 @@ public CommandResult execute(){
 
     //Note: it is better to make the execute() method abstract, by replacing the above method with the line below:
     //public abstract CommandResult execute();
+    
+    public boolean isMutating(){ return false; };
```

Comment:
Good to have comments. this is case this is pretty straight forward but have a brief comment!
Should this have default implementation?